eoscostarica/guide.eoscostarica.io
# feat(docs): add boilerplate content

### GH Issue

Resolves #205  

### Steps to test
1. Check boilerplate content
2. Check is inside to the Developer Guidelines section

#### CheckList
